build-aux/generate.mk

```diff
@@ -366,12 +366,14 @@ $(OSS_HOME)/k8s-config/%/helm-expanded.yaml: \
   $(OSS_HOME)/k8s-config/%/values.yaml \
   $(boguschart_dir)
 	helm template --namespace=$(helm.namespace.$*) --values=$(@D)/values.yaml $(or $(helm.name.$*),$*) $(boguschart_dir) >$@
-
 $(OSS_HOME)/k8s-config/%/output.yaml: \
   $(OSS_HOME)/k8s-config/%/helm-expanded.yaml \
   $(OSS_HOME)/k8s-config/%/require.yaml \
   $(tools/filter-yaml)
 	$(tools/filter-yaml) $(filter %/helm-expanded.yaml,$^) $(filter %/require.yaml,$^) >$@
+k8s-config.clean:
+	rm -f k8s-config/*/helm-expanded.yaml k8s-config/*/output.yaml
+clean: k8s-config.clean
```

Comment:
(This pattern where we have a recipe for something that matches `%.clean`, then explicitly have to do `clean: thing-that-matches-%.clean`, kind of bugged me, given ce9a8d9f1c3cd89ac99e9e9179d4ed755c2c2be. But the alternative:

```
clean: $(patsubst %,%.rm-f,$(wildcard k8s-config/*/helm-expanded.yaml))
clean: $(patsubst %,%.rm-f,$(wildcard k8s-config/*/output.yaml))
```

seems like it might well be worse.)

Reply:
That's bad because it has to do the wildcard at Makefile-parse-time.

Reply:
Yup, won't work. And neither does this:

```
clean:
    rm -rf k8s-config/*/helm-expanded.yaml k8s-config/*/output.yaml
```

...so overall I just kinda have to accept this one. 😉